airbyte-integrations/connectors/source-appsflyer-singer/unit_tests/unit_test.py

```diff
@@ -22,6 +22,5 @@
 SOFTWARE.
 """
 
-
 def test_example_method():
```

Comment:
accidental?

Reply:
i think these are the actual correct style? will make sure to run style check again.